Sources/MintKit/Mint.swift

```diff
@@ -172,7 +173,20 @@ public class Mint {
                     }
                 }
             } catch {
-                throw MintError.repoNotFound(package.gitPath)
+                let metadata = try readMetadata()
+                let linkedExecutables = getLinkedExecutables()
+                let cache = try Cache(path: packagesPath, metadata: metadata, linkedExecutables: linkedExecutables)
+                
+                guard let installedVersions = cache.packages
+                        .first(where: { $0.gitRepo == package.repo })?
+                        .versionDirs.map({ $0.version }),
+                    let fallbackVersion = installedVersions.first else {
```

Comment:
Should it be `.first` or `.last` ?
Maybe best to find latest version possible?

Reply:
Good point. 👍

Reply:
Just to make sure, but it's sorted right?

Reply:
When I tried it was correct!